ansible_collections/arista/avd/roles/eos_designs/templates/underlay/interfaces/ethernet-interfaces.j2

```diff
@@ -34,8 +45,10 @@ ethernet_interfaces:
 {%         endif %}
 {%         if switch.underlay_routing_protocol == "isis" %}
     isis_enable: {{ switch.isis_instance_name }}
-    isis_metric: 50
+    isis_metric: {{ isis_default_metric | arista.avd.default(50) }}
     isis_network_point_to_point: true
+    isis_circuit_type: {{ link.isis_circuit_type | arista.avd.default(isis_default_circuit_type, "level-1-2") }}
```

Comment:
to avoid a lot of changes to existing scenarios and deployments, and since these are default anyway, could we just skip the default "level-1-2" so it will be `none` if nothing is set. Same for hello padding.

Reply:
Btw the `link.isis_circuit_type` and `link.isis_hello_padding` will never be set. They are missing from the underlay logic template (for spine perspective) and from topology fact template.

Reply:
Removed hello padding from this template as you rightfully pointed out there is no knob to set it at present. Also made circuit_type default to none so it doesn't generate so much molecule output for existing scenarios.